tests/torch/test_compile_torch.py

```diff
@@ -603,7 +603,7 @@ def test_compile_torch_or_onnx_activations(
 @pytest.mark.parametrize(
     "model",
     [
-        pytest.param(SimpleQAT),
+        pytest.param(StepFunctionPTQ),
```

Comment:
this was a non-brevitas QAT model for which QAT quantization had to be guessed. we don't support that anymore, so we use ptq here